products/compute/api.yaml

```diff
@@ -31,6 +31,11 @@ objects:
     name: 'Address'
     kind: 'compute#address'
     base_url: projects/{{project}}/regions/{{region}}/addresses
+    list_url: |
+      projects/{{project}}/regions/{{region}}/addresses
+    list_url_response: !ruby/object:Api::Resource::ResponseList
```

Comment:
Was this the intended use of ResponseList? Not entirely sure what it was being used for.

Reply:
I adapted most of this from self_link_query on the sql product. Some of those resources use a list endpoint and then try to match a single thing in the list via some set of identifiers. Db user a good example of this, there is no rest endpoint to retrieve a single user.